README.md

```diff
@@ -61,8 +61,6 @@ may result in the process being killed.
 
 Scout Suite is written in Python and supports the following versions:
 
--   2.7
--   3.4
```

Comment:
We'll need this PR merged before the release, so add a warning about the future deprecation or Python2.7-3.4 instead of removing it now